src/main/java/com/_p1m/productivity_suite/features/note_taking/dto/CreateNoteRequest.java

```diff
@@ -0,0 +1,13 @@
+package com._p1m.productivity_suite.features.note_taking.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CreateNoteRequest {
+    private String title;
+    private String body;
+}
```

Comment:
ဒါက Dto လိုမျိုးဆိုရင်က ပုံမှန်အားဖြင့်

```suggestion
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record CreateNoteRequest(

    @NotBlank(message = "Title must not be blank")
    @Size(max = 100, message = "Title must not exceed 100 characters")
    String title,

    @NotBlank(message = "Body must not be blank")
    @Size(max = 1000, message = "Body must not exceed 1000 characters")
    String body

) {}
```

ကျန်တဲ့ ထည့်ချင်တဲ့ Validation တွေထည့်လို့ရတယ်။

Reply:
ဟုတ်ကဲ့ အစ်ကို